FF1Lib/FF1Rom.cs

```diff
@@ -907,6 +907,8 @@ public void Randomize(Blob seed, Flags flags, Preferences preferences)
 
 			rng = new MT19337(funRngSeed);
 
+			TitanSnack(preferences.TitanSnack, npcdata, rng);

```

Comment:
we reset rng between each Fun% flag so that people racing the same seed get the same fun% results even if they don't have the same preferences, another rng = new MT19337(funRngSeed); should be inserted between TitanSnack and HurrayDwarf

Reply:
ty. done